src/scripts/EasternKingdoms/Karazhan/boss_prince_malchezaar.cpp

```diff
@@ -422,19 +426,18 @@ class boss_malchezaar : public CreatureScript
                     Talk(SAY_AXE_TOSS2);
 
                     Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 0, 100, true);
-                    for (uint8 i = 0; i < 2; ++i)
                     {
```

Comment:
what does this change mean? why removing the ``for`` without removing also its block?